helpers/mail/README.md

```diff
@@ -9,7 +9,7 @@
 Run the [example](https://github.com/sendgrid/sendgrid-go/tree/master/examples/mail) (make sure you have set your environment variable to include your SENDGRID_API_KEY).
```

Comment:
Can you please modify this link to match the helper link below?

Reply:
Sure, miss that. 😬